src/components/Table/TableBody.tsx

```diff
@@ -60,7 +60,7 @@ export default styled(TableBody)`
         : ''}
 
     &.details div {
-      overflow: hidden;
+      /* overflow: hidden; */
```

Comment:
@mvaivre I am not sure if this breaks something, that's why I commented on it to discuss it. I haven't seen any problems, I wonder if you remember why this is here.

Reply:
Yep, if you don't hide the overflow, there's a glitch when closing the details table.

https://user-images.githubusercontent.com/6715943/161777886-8def459c-b498-4484-acd3-0a9f9f0c87ab.mov